docs/monitoring/inventory.md

```diff
@@ -46,21 +43,61 @@ In addition, an overview of all the installed packages on a given system is avai
 - Package name
 - Package version
 
-![Inventory Overview](../images/monitoring/inventory_packages.png)
+#### Comparing Hosts
+
+The Inventory UI allows comparing different systems to quickly and efficiently see differences between them. Use filtering to see differences in specific packages.
```

Comment:
The Inventory UI allows comparing different systems to quickly and efficiently see differences between them. Use filtering to see differences in specific packages.
=>
The Inventory UI let's you compare different systems to quickly see the differences between them and use filtering to see differences between specific packages.